front-end-admin/src/components/donor/DonorList.vue

```diff
@@ -52,6 +55,20 @@ export default {
         name: 'donorInfo',
         params: { id: Number.parseInt(donId) }
       });
+    },
+    formatDate: dateString => {
+      let date = new Date(dateString);
+      return date.toLocaleDateString();
+    },
+    searchTitle() {
+      if (
+        this.searchTerm === 'undefined' ||
+        this.searchTerm.trim().length < 1
+      ) {
+        this.newDependent = '';
+        return;
+      }
+      console.log('search for: ' + this.searchTerm);
```

Comment:
We should probably make both of these functions in a helper class so we don't have to copy them around, but this will work for now

Reply:
Yeah, we need to do that.